src/pages/DisplayCards.tsx

```diff
@@ -1,20 +1,64 @@
 import React, { useState } from "react";
+import ReactTooltip from "react-tooltip";
 import logo from "../assets/images/logo.svg";
+import edit from "../assets/images/edit.svg";
 import "../styles/DisplayCards.css";
 
+export enum CardSide {
+  Left,
+  Right,
+}
+
 const initialValues = {
-  text: "Lorem ipsm dolor sit amet, consectetuer adipiscing elit, sed diam",
+  leftText: "Lorem ipsm dolor sit amet, consectetuer adipiscing elit, sed diam",
+  rightText:
+    "Lorem ipsm dolor sit amet, consectetuer adipiscing elit, sed diam",
+  leftEditing: false,
+  rightEditing: false,
 };
 
 const DisplayCards: React.FC = () => {
-  const [state] = useState(initialValues);
+  const [state, setState] = useState(initialValues);
```

Comment:
I'm sure you can seperate this into using 2 different state hooks for left and right text and name them accordingly. 
type Card {
    readonly text: string;
    readonly isEditing: boolean;
}

You then use ^ that protocol in your code like so
const [ leftCard, setLeftCard] = useState<Card>(initialValueForLeftCard)

Ofc, with appropriate names and not what i used :)

Reply:
Should be better now :) lmk if there's anything else